packages/create-next-app/templates/app-tw/js/eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
+    "plugin:@typescript-eslint/recommended-type-checked",
```

Comment:
Additionally, this doesn't enable our new [_stylistic(-type-checked)_](https://typescript-eslint.io/linting/configs/#stylistic) preset. The stylistic ones are new to typescript-eslint@v6. You can read about them in our announcement post: https://typescript-eslint.io/blog/announcing-typescript-eslint-v6#reworked-configuration-names

Do you all have a preference?

Reply:
I'll just assume that's out of scope. :)